include/otf2xx/attribute_list.hpp

```diff
@@ -526,6 +549,28 @@ namespace detail
             return res;
         }
     };
+
+    template<>
+    struct get_attribute<otf2::definition::attribute::attribute_type::io_file>
+    {
+        template<bool Test = false>
+        void operator()(OTF2_AttributeList* list,
+                        const otf2::definition::attribute& attribute)
+        {
+            static_assert(Test, "Not implemented yet.");
```

Comment:
We need to fix that

Reply:
Okay, this is actually worth a separate issue. There isn't a quick fix for that.